src/invertColor.test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const invertColor = require('./invertColor');
+
+test('#000000', () => {
```

Comment:
Each task case should be testing different scenarios. The description should explain which scenario is being tested. 

Here is an example of good descriptions: https://github.com/mate-academy/js_capitals-first/blob/master/src/capitalsFirst.test.js

Reply:
That's why I wanted guys to automate it 
Not to think about all possible cases

Reply:
C'mon you are a senior engineer. This takes 10 mins.

Reply:
done